.github/workflows/run.yml

```diff
@@ -0,0 +1,48 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          make testsuite.install
+      - name: Test with pytest
+        run: |
+          make testsuite.run
+      - uses: codecov/codecov-action@v2
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          flags: Python-v${{ matrix.python-version }}
+          files: ./coverage.xml
+  package:
+    needs: build
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
Can't we limit the tag pattern?

Reply:
Probably. I'll have to look into it. Can handle on a follow up PR if that's okay. Would you prefer some regex that matched on [0-9].[0-9].[0-9] for example?